lib/stream.js

```diff
@@ -1,25 +1,27 @@
-import {EventEmitter} from 'events'
+import EventEmitter from 'events'
 import {compiler} from './compiler.js'
 import {parser} from './parser.js'
 
 /**
  * @param {import('./compiler.js').Options} [options]
- * @returns {import('stream').Duplex}
+ * @returns {import('events').EventEmitter}
```

Comment:
I don't really understand the original intent of casting `EventEmitter` as `Duplex`.
It either is a `Duplex` in which case `new Duplex()` may make sense for the initializer, or it isn't in which case it may not make sense for the type?

Reply:
The reason the code is the way it is (an instance of EventEmitter, not Duplex) is for a smaller bundle size: such code used to live in `unified` directly and was later externalized to [`unified-stream`](https://github.com/unifiedjs/unified-stream). It’s also in `micromark/stream`

I believe, for Node, readable/writable/duplex are more interfaces (e.g., a `readable: true`) than specifically instances of such classes. And they all inherit from EventEmitter.